.hadolint.yaml

```diff
@@ -0,0 +1,56 @@
+    - DL4004                           # Error: Multiple ENTRYPOINT instructions found
+  warning:
+    - DL3002                           # Warning: Last user should not be root
+    - DL3003                           # Warning: Use WORKDIR to switch to a directory
+    - DL3006                           # Warning: Always tag the version of an image explicitly
+    - DL3007                           # Warning: Using latest is prone to errors
+    - DL3008                           # Warning: Pin versions in apt-get install
+    - DL3013                           # Warning: Pin versions in pip
+    - DL3016                           # Warning: Pin versions in npm
+    - DL3018                           # Warning: Pin versions in apk add
+    - DL3022                           # Warning: COPY --from should reference a previously defined FROM alias
+    - DL3025                           # Warning: Use arguments JSON notation for CMD and ENTRYPOINT arguments
+    - DL3027                           # Warning: Do not use apt; use apt-get or apt-cache instead
+    - DL3030                           # Warning: Use the -y switch to avoid manual input with yum install
+    - DL3045                           # Warning: COPY to a relative destination without WORKDIR set
+    - DL3046                           # Warning: useradd without flag -l and high UID will result in a large image
+    - DL4001                           # Warning: Either use Wget or Curl but not both
+    - DL4003                           # Warning: Multiple CMD instructions found
+    - DL4005                           # Warning: Use SHELL to change the default shell
+    - DL4006                           # Warning: Set the SHELL option -o pipefail before RUN with a pipe in it
+  info:
+    - DL3001                           # Info: Avoid using bash commands like ssh, vim, etc., in Docker
+    - DL3009                           # Info: Delete apt-get lists after installing something
+    - DL3010                           # Info: Use ADD for extracting archives into an image
+    - DL3015                           # Info: Avoid additional packages by specifying --no-install-recommends
+    - DL3019                           # Info: Use --no-cache switch to avoid the need to use --update and remove cache
+    - DL3049                           # Info: Label <label> is missing
+    - DL3050                           # Info: Superfluous label(s) present
+    - DL3059                           # Info: Multiple consecutive RUN instructions. Consider consolidation.
+    - DL3060                           # Info: yarn cache clean missing after yarn install
+  style:
+    - DL3048                           # Style: Invalid Label Key
+strict-labels: true                    # Ensure all required labels are present
+disable-ignore-pragma: false           # Allow ignoring rules with pragmas in Dockerfile
+trustedRegistries:
+  - "docker.io"                        # Trusted registry
```

Comment:
Once confirm with trusted registries with ECR

Reply:
[quay.io](http://quay.io/) is used

Reply:
linter will warn now instead of throwing an error.